build.gradle

```diff
@@ -358,6 +359,7 @@ sourceSets {
     java {
       srcDir 'src/main/java'
       srcDir 'build/generated-sources/jaxb'
+      srcDir('src/main/java/schemas')
```

Comment:
Why a different directory structure this particular jaxb class?  Also its not a schema, the XSD is the schema so if we do want do refactor the directory could do with a better name.